code/modules/events/vent_clog.dm

```diff
@@ -72,3 +72,26 @@
 /datum/round_event/vent_clog/catastrophic
 	randomProbability = 30
 	reagentsAmount = 250
+
+/datum/round_event_control/vent_clog/beer
+	name = "Foamy beer stationwide"
+	typepath = /datum/round_event/vent_clog/beer
+	max_occurrences = 0
+
+/datum/round_event/vent_clog/beer
+	reagentsAmount = 100
+
+/datum/round_event/vent_clog/beer/announce()
+	priority_announce("The scrubbers network is experiencing an unexpected surge of pressurized beer. Some ejection of contents may occur.", "Atmospherics alert")
+
+/datum/round_event/vent_clog/beer/start()

```

Comment:
This entire override could be removed by setting randomProbability to `0` and saferChems to `list("beer")`

Reply:
the override also spawns no cockroaches